puffin/src/cli.rs

```diff
@@ -119,7 +120,8 @@ pub fn main<PB: ProtocolBehavior + Clone + 'static>(
     }
 
     if let Some(_matches) = matches.subcommand_matches("seed") {
-        if let Err(err) = seed(put_registry) {
+        let with_old_obj = true; //with_old_obj = matches.get_flag("o"); // TODO WEIRD error with this: thread 'main' panicked at 'Mismatch between definition and access of `with-old-obj`.
```

Comment:
After this is resolved this can be merged.

Maybe you have to do `get_flag("with-old-obj")`?